torch/_dynamo/variables/torch.py

```diff
@@ -516,6 +516,18 @@ def handle_radians(self, tx: "InstructionTranslator", *args, **kwargs):
                     VariableTracker.build(tx, polyfills.radians), args, kwargs
                 )
 
+        @register(torch.is_inference_mode_enabled)
+        def handle_is_inference_mode_enabled(self, tx: "InstructionTranslator"):
+            unimplemented_v2(
+                gb_type="Encountered torch.is_inference_mode_enabled during tracing",
+                context="",
+                explanation="torch.is_inference_mode_enabled() is not supported",
```

Comment:
on the explanation - our claim is basically that if you are using compile, we want people to use no_grad instead of inference_mode since it gives the same perf under compile and is more composable. Should we mention that in these explanations?

Reply:
Thats a better message. I can do that.

Reply:
Updated the graph break hint messages.